src/System.Private.Interop/src/Internal/Runtime/CompilerHelpers/RuntimeInteropData.CoreRT.cs

```diff
@@ -191,13 +195,18 @@ private unsafe bool TryGetMarshallersForStruct(RuntimeTypeHandle structTypeHandl
             NativeParser entryParser;
             if (TryGetStructData(structTypeHandle, out externalReferences, out entryParser))
             {
-                marshalStub = externalReferences.GetIntPtrFromIndex(entryParser.GetUnsigned());
-                unmarshalStub = externalReferences.GetIntPtrFromIndex(entryParser.GetUnsigned());
-                destroyStub = externalReferences.GetIntPtrFromIndex(entryParser.GetUnsigned());
-                size = (int)entryParser.GetUnsigned();
                 uint mask = entryParser.GetUnsigned();
-                hasInvalidLayout = (mask & 0x1) == 1;
-                return true;
+                if ((mask & 1) != 0)
```

Comment:
Nit: I know this is pre-existing, but since you're adding another bit - the pattern we use in a couple places is to add a file in src/Common/Internal/Runtime that defines the constants/bit shift that we then include from the compiler and the class library. It makes the connection between where this is emitted and where it is used explicit.

Reply:
Done.